common/djangoapps/student/management/commands/bulk_change_enrollment_csv.py

```diff
@@ -42,80 +45,120 @@ def add_arguments(self, parser):
         """ Add argument to the command parser. """
         parser.add_argument(
             '--csv_file_path',
-            required=True,
+            required=False,
             help='Csv file path'
         )
 
     def handle(self, *args, **options):
         """ Main handler for the command."""
         file_path = options['csv_file_path']
 
-        if not path.isfile(file_path):
-            raise CommandError("File not found.")
+        if file_path:
+            if not path.isfile(file_path):
+                raise CommandError("File not found.")
 
-        with open(file_path) as csv_file:
-            course_key = None
-            user = None
-            file_reader = csv.DictReader(csv_file)
-            headers = file_reader.fieldnames
+            with open(file_path, 'rb') as csv_file:
+                self.change_enrollment(csv_file)
 
-            if not ('course_id' in headers and 'mode' in headers and 'user' in headers):
-                raise CommandError('Invalid input CSV file.')
+        else:
 
-            for row in file_reader:
-                try:
-                    course_key = CourseKey.from_string(row['course_id'])
-                except InvalidKeyError:
-                    logger.warning('Invalid or non-existent course id [{}]'.format(row['course_id']))
+            csv_file = BulkUpdatenrollmentConfiguration.current().csv_file
+            self.change_enrollment(csv_file)
 
+    def change_enrollment(self, csv_file):
```

Comment:
nit: As we are modifying more that one objects, the name should be plural `change_enrollments`.